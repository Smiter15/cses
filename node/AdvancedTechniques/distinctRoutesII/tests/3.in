200 1000 5
196 120
125 26
28 50
70 5
8 27
149 200
86 58
138 29
118 18
48 123
145 28
92 183
112 94
147 162
153 149
129 149
68 5
174 47
54 43
52 61
22 152
12 107
9 101
87 78
119 134
200 15
162 43
197 73
89 95
186 41
44 66
141 63
70 66
110 75
61 113
87 34
64 98
140 24
151 178
102 69
111 53
83 24
96 31
48 184
125 33
176 120
98 69
183 167
134 44
193 194
156 79
27 171
129 118
194 79
147 158
148 181
133 195
163 181
130 87
13 57
159 63
125 141
181 11
184 174
4 91
52 183
61 132
52 54
10 99
101 108
8 200
88 140
194 2
120 75
89 85
131 47
147 56
139 127
66 74
100 21
164 104
155 176
35 7
181 197
57 81
104 64
32 144
3 177
145 123
51 185
159 182
183 105
189 73
183 54
146 35
26 61
138 181
36 124
139 169
72 147
1 176
10 33
71 164
9 167
38 60
104 196
41 86
100 29
111 83
34 56
117 11
69 6
143 104
81 33
24 153
179 25
80 140
57 102
104 17
68 141
76 171
89 68
9 140
48 119
22 33
126 90
44 194
47 138
50 80
171 73
32 101
97 63
41 105
164 96
74 183
188 149
106 76
169 114
16 168
184 189
153 61
31 148
55 77
85 56
156 7
80 139
108 129
1 100
33 116
195 106
99 182
7 4
10 127
69 177
46 4
165 198
64 147
145 70
176 182
77 83
188 157
22 95
85 174
194 69
42 140
127 92
74 85
95 83
11 115
11 8
70 127
6 133
74 102
155 49
83 31
131 23
187 104
69 23
30 15
92 71
154 170
162 27
54 8
61 16
74 23
23 143
150 180
27 23
34 133
23 116
116 153
90 54
9 159
77 38
173 120
60 78
89 173
57 94
132 69
61 157
92 83
99 41
15 77
63 48
162 37
115 77
189 5
130 149
115 158
138 87
29 149
164 69
121 172
162 4
41 22
160 132
57 10
172 23
87 186
70 200
22 136
76 162
92 157
111 6
67 200
44 95
149 160
164 21
131 125
25 171
99 184
54 107
194 25
129 32
162 150
31 183
85 58
97 177
166 159
134 32
158 130
79 58
89 42
79 171
118 55
94 16
38 147
122 183
172 2
14 85
151 196
20 36
5 116
164 171
26 49
154 135
178 32
193 18
25 23
189 143
77 34
110 66
180 32
49 103
33 62
188 136
10 7
171 14
74 86
136 51
134 90
185 64
73 129
33 98
151 21
192 179
6 101
45 186
84 93
88 69
106 27
4 115
185 174
81 10
11 16
60 96
57 45
159 130
173 192
179 76
164 170
54 170
70 50
107 154
142 194
22 200
173 25
114 157
150 194
167 112
162 26
7 18
22 166
147 195
109 77
22 24
46 75
193 116
77 51
21 18
197 194
18 83
77 119
28 43
30 189
21 188
159 42
153 105
188 13
134 9
37 179
36 181
46 112
32 66
81 180
59 50
148 28
84 24
98 63
19 79
4 191
67 162
184 171
18 172
184 151
132 172
82 119
104 127
182 135
145 15
66 3
16 13
183 51
189 80
113 159
159 68
149 177
181 132
147 151
55 130
101 187
53 134
57 157
61 191
43 5
5 109
159 172
121 93
22 61
125 86
19 35
106 167
11 86
51 157
36 105
87 89
43 139
22 100
95 34
190 46
198 101
17 118
193 190
131 117
75 149
103 180
68 113
18 118
22 108
93 17
198 67
116 45
52 157
134 42
154 131
94 122
200 38
111 130
107 137
31 9
69 98
122 86
167 191
90 140
98 135
156 95
153 23
139 4
179 72
197 132
154 90
5 154
128 115
83 34
52 65
42 120
162 115
157 80
105 46
20 154
132 175
1 84
150 91
88 128
115 36
12 68
126 16
21 5
83 56
190 134
120 31
5 34
98 161
65 2
158 84
108 90
11 108
173 160
198 12
39 112
48 2
132 173
18 152
129 52
159 118
185 31
100 141
64 197
101 128
151 58
164 58
99 126
165 72
85 114
83 75
178 130
77 109
54 46
12 134
97 171
51 81
1 103
173 16
99 73
95 49
82 58
106 181
135 22
4 19
136 86
94 65
168 122
167 184
107 63
106 109
95 58
11 67
73 110
91 120
110 172
129 168
189 107
34 76
145 127
140 64
48 64
36 14
48 94
28 105
155 122
163 165
188 35
26 78
154 88
2 83
55 105
98 195
164 131
71 108
58 162
123 57
26 158
23 25
4 50
129 110
124 91
26 111
126 4
141 31
91 142
65 97
46 162
91 161
93 161
68 102
75 190
110 100
100 84
124 55
134 52
164 57
143 141
119 20
144 59
50 34
200 67
130 198
28 21
158 72
133 178
125 7
119 35
58 170
118 181
176 3
185 91
11 127
32 130
98 175
140 99
165 90
89 145
62 198
16 147
127 141
127 163
167 44
95 143
199 79
33 9
157 16
191 30
105 151
121 69
40 9
165 54
158 133
200 85
25 56
82 151
170 48
146 65
142 144
95 127
153 52
57 132
23 161
34 143
182 74
103 146
79 13
78 30
97 102
85 178
52 55
177 150
71 119
93 165
168 138
191 25
22 13
137 55
108 70
80 162
16 158
174 45
139 22
147 24
199 150
30 128
99 100
198 144
64 194
140 32
131 130
8 104
95 96
159 154
166 190
115 181
148 41
177 86
62 123
61 13
25 90
140 75
189 137
18 5
130 110
36 12
39 195
116 85
90 26
38 7
15 105
139 114
145 116
152 158
190 12
170 14
54 109
41 42
197 10
46 126
119 33
90 110
179 157
194 21
182 184
27 148
88 166
62 184
81 13
141 22
176 111
36 29
171 28
23 168
37 20
45 108
53 86
17 183
42 162
3 175
145 30
75 32
147 88
167 27
100 108
48 27
170 35
56 92
14 30
11 3
49 8
110 101
93 126
36 87
87 38
13 32
67 87
15 13
126 182
41 191
4 162
177 11
53 143
40 16
145 88
23 54
141 67
179 176
105 17
13 70
200 32
108 167
196 34
100 139
10 140
53 57
23 57
199 62
72 95
102 46
148 42
113 78
17 169
95 181
38 46
12 146
12 8
179 111
146 140
2 178
153 6
34 127
63 185
141 86
79 42
132 179
35 196
56 51
164 102
31 154
100 195
99 111
154 29
174 51
168 127
28 72
133 13
199 147
197 91
32 28
132 163
158 166
158 165
168 74
67 150
27 166
144 77
28 97
57 153
147 176
80 174
200 164
187 198
110 135
40 141
114 171
56 23
88 165
188 7
38 54
156 16
171 100
44 150
148 100
103 47
28 134
172 27
97 110
176 6
197 15
98 110
137 153
133 24
70 72
30 107
125 137
75 47
188 118
148 136
170 50
182 197
23 31
15 198
188 11
12 86
72 148
190 156
123 59
27 104
118 103
40 192
101 28
61 49
115 30
50 48
178 185
152 200
15 166
197 3
128 160
88 200
59 29
155 181
88 134
84 157
36 76
148 137
103 160
185 17
142 186
16 119
170 167
119 185
85 67
139 12
179 178
163 37
6 118
18 93
3 53
49 38
128 148
60 110
150 1
48 165
71 17
84 141
153 168
8 91
179 186
68 142
174 74
11 191
118 13
196 119
2 123
18 128
57 29
189 6
29 36
123 52
53 6
192 5
125 151
48 55
88 115
148 62
140 158
4 58
122 199
128 76
74 97
3 130
126 143
87 106
179 161
157 183
118 101
139 42
51 186
97 139
53 44
152 192
154 155
1 27
154 128
102 179
177 56
108 102
199 146
7 129
59 116
63 176
16 1
18 66
156 191
109 107
182 62
14 183
89 119
62 142
97 38
198 156
91 148
53 98
76 111
165 138
87 180
90 56
197 28
169 105
18 29
163 152
91 176
134 131
94 147
24 142
137 21
97 27
120 150
124 147
60 40
84 98
19 126
153 16
153 73
120 123
102 82
74 144
26 55
105 49
197 41
70 93
194 143
88 6
183 171
55 153
48 8
168 1
80 39
123 133
73 196
76 34
154 101
88 58
150 84
51 171
111 122
12 198
79 194
121 134
147 120
141 66
120 172
57 194
141 132
114 94
185 1
104 86
37 154
8 165
23 70
38 34
20 117
180 92
50 78
11 142
95 131
65 103
76 121
135 17
5 48
62 141
1 117
26 37
22 185
153 44
54 190
30 73
106 161
164 107
190 175
200 84
57 65
139 84
185 73
73 114
81 25
30 58
13 52
42 165
47 96
154 28
120 12
188 84
118 177
10 118
13 18
134 103
101 70
127 1
11 199
66 43
23 95
101 35
98 142
151 145
189 105
124 109
116 112
144 185
40 153
102 4
106 22
105 33
174 67
183 39
109 158
144 183
103 95
79 43
151 3
103 66
114 46
50 105
150 104
109 71
138 166
93 194
150 66
60 85
105 169
42 190
129 137
172 183
31 51
26 133
5 80
198 118
126 200
174 75
103 34
98 192
152 190
138 101
55 197
137 111
117 130
100 198
67 24
132 49
26 157
182 30
37 131
99 114
130 72
