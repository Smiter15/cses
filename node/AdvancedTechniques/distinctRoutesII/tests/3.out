21
6
1 176 182 135 22 200 
5
1 100 198 67 200 
5
1 84 93 126 200 
5
1 27 23 70 200 
5
1 117 130 149 200 
